packages/lesswrong/components/comments/CommentsNode.tsx

```diff
@@ -139,7 +139,7 @@ const CommentsNode = ({
 
   const {hash: commentHash} = useLocation();
   useEffect(() => {
-    if (comment && commentHash === ("#" + comment._id) && post && postPage) {
```

Comment:
This is all that was required to make the comments scroll into view. This will affect other pages (e.g. user profiles, tag discussion) but:
 - for most of these pages we don't make links with comment hashes anyway
 - the effect should be to make scroll into view work for those pages, so this should be good anyway

Reply:
This actually wound up being a bug in the context of side-comments; if you're on a comment-hash URL and you hover over the hash-linked comment, it scrolls down to where that comment appears in the comments list, ie, scrolls away from the side-comment you were looking at.